### PR TITLE
fix(lsp): make failed to load config error descriptive

### PR DESCRIPTION
**Pull request by satyarohith**
Before:
```
Failed to load configuration file
```
After:
```
Could not find the config file: /Users/sr/p/deno_lsp/tsconfig.json
```

Closes https://github.com/denoland/deno/issues/10646

### CODE REVIEW

File: cli/lsp/language_server.rs [hunk 1]
```diff
@@ -432,8 +431,7 @@ impl Inner {
         ))
       }?;
 
-      let config_file = ConfigFile::read(config_url.path())
-        .context("Failed to load configuration file")?;
```
Comment (satyarohith) [inline cli/lsp/language_server.rs:436]: The error from ConfigFile::read is already descriptive. 
```
Updating TypeScript configuration from: "./tsconfig.json"
Updating tsconfig has errored: Could not find the config file: /Users/sr/p/deno_lsp/tsconfig.json
```

**Comment by ry** [comment]
Could you print the error messages before and after? I'm having trouble following what has changed.

**Comment by satyarohith** [comment]
> Could you show us the error messages before and after? I'm having trouble following what has changed.

Previous:
```
Failed to load configurations file
```
Now:
```
Could not find the config file: /Users/sr/p/deno_lsp/tsconfig.json
```

**Comment by ry** [comment]
@satyarohith Would it be difficult to add a test which asserts the filename is part of the error message?

**Review by ry — APPROVED** [review]
LGTM - nice test

**Pull request merged**